### PR TITLE
Documentation cleanup

### PR DESCRIPTION
**Pull request by huikang**
- use chassis consistently
- run as root user for first time user

Signed-off-by: Hui Kang kangh@us.ibm.com

### CODE REVIEW

**Comment by huikang** [comment]
partially address #119

File: ansible/README.rst [hunk 1]
```diff
@@ -27,25 +27,32 @@ Host machine requirements
...
-- 2 OVN chassis host to run emulated OVN chassis container
+- 2 hosts to run emulated OVN chassis container
 - 1 host to run rally container (could be the same as the OVN database node)
 
 .. image:: ovn-emulation-deployment.png
    :alt: OVN emulation deployment
 
-  
+
+.. note:: The above setting is just an example. For example, all OVN emulated chassis could be run on one host. Further, an all-in-one configuration deploys all containers on a single host.
```
Comment (mestery) [inline ansible/README.rst]: Can you linewrap this file at 80 characters please?

File: ansible/README.rst [hunk 2]
```diff
@@ -27,25 +27,32 @@ Host machine requirements
...
+
 Installing Dependencies
 -----------------------
 
+
```
Comment (mestery) [inline ansible/README.rst]: Why an extra space here and on line 40?

File: ansible/README.rst [hunk 3]
```diff
@@ -27,25 +27,32 @@ Host machine requirements
...
 The deploy node needs ansible. All the other nodes should have docker engine
 installed. The following python package are required on all the nodes as well.
 
 ::
 
     pip install -U docker-py netaddr
 
+
+.. note:: If you are a first time user, please run all the following commands as ``root`` user literally, not sudo and set ``deploy_user: "root"`` in ``ansible/group_vars/all.yml``. As getting familiar with the framework, variable ``deploy_user`` can be configured.
```
Comment (mestery) [inline ansible/README.rst]: Linewrap at 80 characters.

File: ansible/README.rst (hunk 2 above)
Comment (huikang) [reply ansible/README.rst]: will remove it. Thanks.

File: ansible/README.rst (hunk 1 above)
Comment (huikang) [reply ansible/README.rst]: Thanks for reviewing the PR, @mestery !

When the line is wrapped at 80 chars, the file could not be rendered correctly as an online rst file, especially for some rst symbols e.g., the above .. note::. Do you think we need to remove all rst symbols in this README file?

Comment (huikang) [reply ansible/README.rst]: I find out how to wrap line in rst. I will update the PR.

File: ansible/README.rst [hunk 4]
```diff
@@ -73,6 +85,25 @@ The remaining of this guide uses OVN-SCALE-TEST as the image name. You need to
 change the name in your deployment.
 
 
+Build upstream OVS/OVN images
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The pre-built images stored on Dockerhub may not include the latest OVS commit.
+To test latest OVS/OVN, the following steps need to be run
+
+1. Edit ``ansible/docker/ovn/Makefile`` and run ``make`` to rebuild the image. In ``ansible/docker/ovn/Makefile``, the variable ``image_name`` should be set appropriately, e.g.,
```
Comment (mestery) [inline ansible/README.rst]: I'm sorry to keep nit-picking, but I really think these should be line wrapped at 80 characters. :)

**Comment by mestery** [comment]
Once you line-wrap the last line I commented on, I will merge this. Thanks @huikang!

**Comment by huikang** [comment]
@mestery thanks for your review.

**Pull request merged**